src/main/java/fr/insee/pearljam/domain/surveyunit/model/IdentificationState.java

```diff
@@ -116,6 +113,11 @@ private static IdentificationState houseF2F(Identification identification) {
             return IdentificationState.FINISHED;
         }
 
+        if (identification.access() == AccessQuestionValue.NACC &&
+                identification.identification() == IdentificationQuestionValue.IDENTIFIED) {
+            return IdentificationState.FINISHED;
+        }
+
```

Comment:
should be moved to line #92-93
identification.identification() == IdentificationQuestionValue.IDENTIFIED is not necessary (value already checked)
=> early returning ONGOING case for clarity